index.html

```diff
@@ -774,8 +764,8 @@ <h6 id="sec-provisioning-profile-id">Provisioning Profile IDs</h6>
         </section>
         <section  id="sec-signature-pkcs7">
             <h3>PKCS#7 Signature Scheme</h3>
-            <p>This signature scheme is based on the PKCS #7 Cryptographic Message Syntax [[?rfc2315]]. This method admits recursion, so one "digital envelope", the data block including encrypted content and encrypted keys, can be nested inside other. This signature scheme MAY cover also the different stages of packaging signature described in <a href="#sec-signature-rpk">RPK Signature Scheme</a>, including the developer signature and distributor signature.</p>
-            <p>This signature scheme includes the provisioning profile of the MiniApp at the developer signing stage. At a second stage, a distributor MAY replace the developer's signature with the its own signature. Likely in the <a href="#sec-signature-rpk">RPK Signature Scheme</a>, this approach generates a Signing Block to be included into the package file, according to the <a href="sec-miniapp-digital-signature-requirements">Digital Signature Requirements</a>.</p>
+            <p>This signature scheme is based on the PKCS #7 Cryptographic Message Syntax [[?rfc2315]]. It MAY cover different stages of packaging signature described in <a href="#sec-signature-rpk">RPK Signature Scheme</a>, including the developer signature and distributor signature.</p>
+            <p>This signature scheme includes the [=Provisioning Profile=] of the MiniApp and the developer's signature at the first stage (developer signing stage). At the second stage (distributor signing stage), a distributor MAY replace the developer's signature with the its own signature. Like the <a href="#sec-signature-rpk">RPK Signature Scheme</a>, this approach generates a Signing Block to be included into the package file, according to the <a href="sec-miniapp-digital-signature-requirements">Digital Signature Requirements</a>.</p>
```

Comment:
The `#` is missing in the URL fragment:
`<a href="#sec-miniapp-digital-signature-requirements">Digital Signature Requirements</a>`